beamline-environments/collection-dev/meta.yaml

```diff
@@ -0,0 +1,22 @@
+package:
+  name: collection-dev
+  version: '2015-03'
+# https://github.com/ericdill/conda-prescriptions/blob/add-dp-v0.2.2/beamline-environments/proposed-deployment-strategy.rst
+
+
+build:
+  number: 1
+
+requirements:
+  run:
+    - openpyxl
+    # sort of our deps
+    - pyolog >=v4.0.0.post5+g1a24409
+    - channelarchiver >=v0.0.5.post6
+    - pyepics >=v3.2.4.post19+g763d30
```

Comment:
I guess we should do this pinning in ophyd so that if some does `conda install ophyd` the correct version of pyepics is installed.